src/edge/simulator.ts

```diff
@@ -56,10 +63,14 @@ export class Simulator {
     private static learnMoreUrl = "https://aka.ms/AA3nuw8";
 
     private static async validateSimulatorInstalled(outputChannel: vscode.OutputChannel = null): Promise<InstallReturn> {
+        const telemetryName = "simulatorInstalled";
         if (await Simulator.simulatorInstalled()) {
             return InstallReturn.Success;
         } else {
-            return await Simulator.installSimulatorWithPip(true, Constants.needSimulatorInstalledMsg, outputChannel);
+            TelemetryClient.sendEvent(`${telemetryName}.install`);
+            const installRes = await Simulator.installSimulatorWithPip(true, Constants.needSimulatorInstalledMsg, outputChannel);
+            TelemetryClient.sendEvent(`${telemetryName}.${InstallReturn[installRes]}`);
```

Comment:
${telemetryName}.install.${InstallReturn[installRes]} ?

Reply:
telemetryName is already contains install. It only sent when install needed.

Reply:
will update to be consistent with update scenario, Thx

Reply:
So ``${telemetryName}.install`` could be ``${telemetryName}``?

Reply:
updated.